[faudio] update to 25.04

rkitover opened the pull request:
- [ ] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md).
- [x] SHA512s are updated for each updated download.
- [ ] The "supports" clause reflects platforms that may be fixed by this new version.
- [ ] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [ ] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

rkitover committed:
[faudio] update to 25.04

Signed-off-by: Rafael Kitover <rkitover@gmail.com>

LilyWangLL commented on ports/faudio/portfile.cmake:
```diff
@@ -21,7 +21,10 @@ vcpkg_cmake_install()
 vcpkg_fixup_pkgconfig()
 
-file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include" "${CURRENT_PACKAGES_DIR}/debug/share")
+file(REMOVE_RECURSE
+    "${CURRENT_PACKAGES_DIR}/debug/include"
+    "${CURRENT_PACKAGES_DIR}/debug/share"
+)
 
 vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/FAudio)
```
```suggestion
vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/FAudio)

file(REMOVE_RECURSE
    "${CURRENT_PACKAGES_DIR}/debug/include"
    "${CURRENT_PACKAGES_DIR}/debug/share"
)

```

LilyWangLL replied:
This will fix the pipeline failures.

rkitover replied:
Applied, thank you!

LilyWangLL committed:
[faudio] fix cleanup in portfile

Fix sequence of commands to delete the directories that need to be                                                                      deleted for the port to install.

Signed-off-by: Rafael Kitover <rkitover@gmail.com>

rkitover commented:
CI passes now, thank you very much @LilyWangLL !
